packages/core/src/components/DataTable/helpers/expandData.ts

```diff
@@ -1,11 +1,13 @@
 import { SortDirectionType } from 'react-virtualized';
-import sortList from './sortData';
-import { IndexedParentRow, IndexedChildRow, ExpandedRow } from '../types';
+import sortData from './sortData';
+import { SelectedRows, IndexedParentRow, IndexedChildRow, ExpandedRow } from '../types';
 
 /*  Iterate over the sortedDataList in state to flatten out children stashed in metadata */
 export default function expandData(
   sortedData: IndexedParentRow[],
```

Comment:
Not sure for this PR, but this function has far too many arguments. Maybe it should be a single object argument with multiple properties.